vowpalwabbit/core/include/vw/core/array_parameters_dense.h

```diff
@@ -9,6 +9,11 @@
 #include <cassert>
 #include <cstdint>
 #include <iterator>
+namespace VW
+{
+
+namespace details
```

Comment:
I assume the dense parameters/iterator should never be used outside of VW code?

Reply:
The dense_parameters::iterator typedef is the intended way to access the type